app/controllers/panel_controller.rb

```diff
@@ -59,4 +59,27 @@ def panel_page
     query_params = request.query_parameters.except(:id)
     redirect_to panel_index_path(panel_id: panel_with_panel_page, anchor: panel_page_id, **query_params)
   end
+
+  private def cronjob_from_params
+    cronjob_class_name = params.require(:cronjob_class_name)
+    Object.const_get(cronjob_class_name)
```

Comment:
Again, I am not too happy about `const_get` because of injection. You can take most of the functions here (at least `run` and `reset`, along with a `cronjob_from_name` helper) and move them to a separate `lib` module similar to `ResultsValidators::Utils`

Reply:
During the course of the review, I doscovered that there is already `JobUtils`, so please use that!